falcon/routing/compiled.py

```diff
@@ -15,12 +15,14 @@
 """Default routing engine."""
 
 from collections import UserDict
+from inspect import iscoroutinefunction
 import keyword
 import re
 import textwrap
 
 from falcon.routing import converters
```

Comment:
We'll probably need to document that converters are always sync since they are supposed to do simple path parsing and validation (as opposed to I/O).

Reply:
👍 I've added this to my list of things to document.